platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/cli/CompareStatesCommand.java

```diff
@@ -113,18 +118,23 @@ private void setDeepComparison(final boolean deepComparison) {
     /**
      * Load a state from disk and hash it.
      *
-     * @param statePath
-     * 		the location of the state to load
+     * @param statePath the location of the state to load
      * @return the loaded state
      */
-    private static SignedState loadAndHashState(final Path statePath) throws IOException {
+    private static ReservedSignedState loadAndHashState(final Path statePath) throws IOException {
         System.out.println("Loading state from " + statePath);
-        final SignedState signedState =
-                SignedStateFileReader.readStateFile(statePath).signedState();
+
+        final PlatformContext platformContext = new DefaultPlatformContext(
```

Comment:
I've moved the creation of the context up into the `call()` method, so at least now we aren't creating two of them (since we are loading 2 states).

It's hard for me to figure out an elegant way to move it higher in the call stack. Some commands don't use a context at all, and other such as `pcli browse` (which calls into `Browser.java`) need to create the context using a completely different strategy.